app.js

```diff
@@ -38,7 +38,7 @@ hamburger.addEventListener('click', function () {
         each.style.textDecoration = 'none';
         each.style.color = 'white';
         each.style.fontWeight = '700';
-        each.style.fontSize = '2.5rem';
+        each.style.fontSize = '2.0rem';
        })
 
 
```

Comment:
## Participant Section
![image](https://user-images.githubusercontent.com/93039359/215686156-b2c424a4-1fe3-49f1-9ed2-83fc599485d7.png)
## Commits
![image](https://user-images.githubusercontent.com/93039359/215686248-66ee7579-7e9f-437d-8037-47185dbe05cf.png)

- [ ] In this project, all members of your team should contribute to all the repositories means your partner will contribute to your pull request and you will contribute to your partner's repository :+1: Kindly ask your partner to make some contribution in this pull request :blush: